app/src/main/java/ayds/winchester/songinfo/moredetails/presentation/MoreDetailsPresenter.kt

```diff
@@ -0,0 +1,45 @@
+package ayds.winchester.songinfo.moredetails.presentation
+
+import ayds.observer.Observable
+import ayds.observer.Subject
+import ayds.winchester.songinfo.moredetails.domain.entity.Info.ArtistInfo
+import ayds.winchester.songinfo.moredetails.domain.entity.Info.EmptyInfo
+import ayds.winchester.songinfo.moredetails.domain.repository.WikipediaRepository
+
+interface MoreDetailsPresenter {
+    val uiStateObservable: Observable<MoreDetailsUiState>
+    fun createThread(artistName: String)
+}
+
+internal class MoreDetailsPresenterImpl(private val wikipediaRepository: WikipediaRepository, private val infoDescriptionHelper: InfoDescriptionHelper): MoreDetailsPresenter{
```

Comment:
Fijense de indentarlo mejor. Podria ser 
```suggestion
internal class MoreDetailsPresenterImpl(
   private val wikipediaRepository: WikipediaRepository, 
   private val infoDescriptionHelper: InfoDescriptionHelper
): MoreDetailsPresenter {
```